.github/workflows/fly-deploy.yml

```
@@ -9,6 +9,7 @@ jobs:
   deploy:
     name: Deploy app
     runs-on: ubuntu-latest
+    if: false  # Temporarily disabled - remove this line to re-enable
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)